### PR TITLE
Feature/#130 cleaup boilerplate

### LINKED ISSUE #130: Cleanup usually unused boiler plate with strip script
Listing of items which should be stripped by the project tool script (at least with a yes/no)

- [x] Remove Todo database and related models / replace with something chosen and easily configured (just DB, no models perhaps)
- [x] Remove Todo screens and related code
- [ ] Remove dummy API
- [x] Remove TODO api

This list is not exhaustive (yet)

### PR DESCRIPTION
**Pull request by vanlooverenkoen**
Fixed #130

### CODE REVIEW

**Comment by NicolaChimerapps** [comment]
Thanks, will take a look after the holidays

**Comment by NicolaChimerapps** [comment]
Please also update flutter_test to latest nullsafety release.

File: tool/dart_tool/strip_boilerplate_project.dart [hunk 1]
```diff
@@ -0,0 +1,251 @@
+import 'dart:io';
+
+void main(List<String> args) {
+  bool keepTestDir;
+  if (args.isEmpty || (args[0] != 'true' && args[0] != 'false')) {
+    do {
+      final result = stdin.readLineSync();
```
Comment (NicolaChimerapps) [inline tool/dart_tool/strip_boilerplate_project.dart:7]: Inform the user he needs to enter something...

File: tool/dart_tool/strip_boilerplate_project.dart [hunk 2]
```diff
@@ -0,0 +1,251 @@
+import 'dart:io';
+
+void main(List<String> args) {
```
Comment (NicolaChimerapps) [inline tool/dart_tool/strip_boilerplate_project.dart:3]: We need to inform the user of the order in which shells need to be run. It seems for example install_fvm.sh is required before running this.
Alternatively, run the required scripts within these

File: tool/dart_tool/strip_boilerplate_project.dart [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+  executeCommand('fvm', ['flutter', 'packages', 'get']);
+  executeCommand('fvm', ['flutter', 'packages', 'pub', 'run', 'build_runner', 'build', '--delete-conflicting-outputs']);
+}
+
+void executeCommand(String cmd, List<String> params) {
+  final fullCommand = '$cmd ${params.join(' ')}';
+  try {
+    Logger.debug('\nExecuting command:\n$fullCommand');
+    final result = Process.runSync(cmd, params);
+    final dynamic output = result.stdout;
```
Comment (NicolaChimerapps) [inline tool/dart_tool/strip_boilerplate_project.dart:46]: Seems like this output is not captured/returned. Failures also just keep running next steps

**Review by NicolaChimerapps — CHANGES_REQUESTED** [review]

**Review by NicolaChimerapps — DISMISSED** [review]

**Review by NicolaChimerapps — DISMISSED** [review]

**Review by HannesVDB — DISMISSED** [review]

**Review by NicolaChimerapps — APPROVED** [review]

**Pull request merged**